dev/linters/logging/logging.go

```diff
@@ -0,0 +1,52 @@
+package logging
+
+import (
+	"fmt"
+
+	"github.com/OpenPeeDeeP/depguard/v2"
+	"golang.org/x/tools/go/analysis"
+
+	"github.com/sourcegraph/sourcegraph/dev/linters/nolint"
+)
+
+// This analyzer is modeled after the one in
+// https://sourcegraph.sourcegraph.com/github.com/sourcegraph/sourcegraph@f6ae87add606c65876b87d378929fcb80c3bb493/-/blob/dev/linters/depguard/depguard.go
+// These could potentially be combined into one analyzer.
+var Analyzer *analysis.Analyzer = createAnalyzer()
+
+const USE_LOG_INSTEAD = `use "github.com/sourcegraph/log" instead`
```

Comment:
```suggestion
const UseLogInsteadMessage = `use "github.com/sourcegraph/log" instead`
```
In Go constants are preferred to by CamelCased